website_user_login_redirect_custom/models/res_config_settings.py

```
@@ -0,0 +1,32 @@
+        help="If checked, users will be redirected to a custom URL after login/signup.",
+    )
+
+    website_login_redirect_url = fields.Char(
+        string="Custom URL",
+        config_parameter="website_user_login_redirect_custom.url",
+        help="Relative URL path where users will be redirected after login/signup. "
+        "Example: '/', '/shop', '/thank-you-for-joining-us'.",
+    )
+
+    @api.constrains("website_login_redirect_url")
+    def _check_url_format(self):
```

Ruff (0.14.0)

Missing return type annotation for private function `_check_url_format`

Add return type annotation: `None`

(ANN202)